hypervisor/src/mshv/x86_64/mod.rs

```diff
@@ -43,6 +43,10 @@ pub struct VcpuMshvState {
     pub vp_states: AllVpStateComponents,
 }
 
+#[derive(Clone, Copy, Debug, Default, Serialize, Deserialize)]
+pub struct MshvClockData {
+    pub ref_time: u64,
+}
```

Comment:
nit: I think there should be a whitespace line here.

Reply:
Fixed.